### PR TITLE
Izio38 patch 1

### PR DESCRIPTION
**Pull request by izio38**
This work around worked for me.
I needed to wait a bit more to set the voice proximity

### CODE REVIEW

**Comment by izio38** [comment]
Actually this isn't a debugging since this wasn't a bug ahah.
But I'm glad if I can help :)

File: client/main.lua [hunk 1]
```diff
@@ -37,8 +37,11 @@ AddEventHandler('onClientMapStart', function()
 	end
 end)
 
+AddEventHandler('playerSpawned', function()
+	NetworkSetTalkerProximity(voice.default)
```
Comment (ElPumpo) [inline client/main.lua]: Why reset to default voice proximity? That doesn't always match up with current voice proximity!

**Comment by izio38** [comment]
because playerSpawn event is triggered onClientMapStart which is called only once by connection? --'

**Comment by ElPumpo** [comment]
You're incorrect. `playerSpawned` triggers everytime you respawn

**Pull request merged**